MonstieWash/Assets/Scripts/Traversal/GameSceneManager.cs

```diff
@@ -261,9 +290,9 @@ public async void GoToMainMenu()
         await UnloadActiveLevelScenes();
 
         GetComponentInChildren<MusicManager>().SetMusic(MusicManager.MusicType.Morning);
-        InputManager.Instance.SetCursorMode(false);
-        InputManager.Instance.SetControlScheme(InputManager.ControlScheme.MenuActions);
-        MoveToScene(levelSelectScene.SceneName);
+        InputManager.Instance.SetCursorMode(true);
+        InputManager.Instance.SetControlScheme(InputManager.ControlScheme.PlayerActions);
```

Comment:
Might be a design decision from the higher ups (if so ignore this) but using right-click to navigate scenes is a little unintuitive. Most scenes use right-click to navigate to/from using buttons but the level select and upgrade menus both use left click (inconsistent).

Reply:
This is v true, I'll defs have to follow up with the designers on this one coz it can feel a bit weird changing up the controls so often going around the bedroom.

Reply:
actually nvm I'll just make both menus navigate with the same button (on controller - can't really do it on mouse yet unfortunately)